pangeo-cnes.tex

```diff
@@ -83,75 +99,94 @@ \subsection{Motivations, mission and goals}
+community-driven project, in the model of successful open-source software like
+Linux, Python, and Jupyter.  In this spirit, all Pangeo discussions and products
+occur on GitHub\cite{b3}, where anyone can easily get involved in the community.
+As of today, Pangeo spans a list of people from different governmental agencies,
+universities, or private companies, and from different countries (the USA, the
+UK, France, Austalia to name a few).  By making the collaboration as broad as
+possible, Pangeo successfully leverages shared expertise to accomplish things no
+institution could alone.
 
 
-\subsection{Technology stack}
+\subsection{Technology contributions to the Scientific Python stack}
 \label{ssec:techstack}
 
-Pangeo software ecosystem fits directly in the Scientific Python stack, involving well known packages such as Numpy, Pandas, or Sickit-learn.
-
-%\begin{figure}
-%  \centering
-%  \includegraphics[width=\columnwidth]{pangeo_python_stack.png}
-%  \caption{\label{scipy_stack} Pangeo Scientific Python ecosystem.}
-%\end{figure}
-
-Three Python software projects are at the core of Pangeo:
+Pangeo's software ecosystem fits directly into the Scientific Python stack, involving packages such as Numpy, Pandas, or Sickit-learn.  Three Python software projects are at the core of Pangeo:
 \begin{itemize}
-\item Jupyter: Jupyter notebooks, Jupyter Lab, and JupyterHub enable interactive computing and analysis from a web browser and multiple user support. They are quickly becoming the standard open-source format for interactive computing in Python, as well as other languages such as Julia and R.
-\item Dask is a library for parallel computing that coordinates with Python’s existing scientific software ecosystem, including libraries cited above. In many cases, it offers users the ability to take existing workflows and quickly scale them to much larger applications. Dask-distributed is an extension of Dask that facilitates parallel execution across many computers.
-\item Xarray is the interface for working with big datasets: it offers a Pandas like API for dealing with labelled n-dimensionnal arrays with a self-describing scientific data capability and strong backends to standard and upcoming community data file formats and access (e.g. netCDF, GeoTIFFs, OPeNDAP).
+\item Dask is a library for parallel and distributed computing that coordinates with Python’s existing scientific software ecosystem.  In many cases, it offers users the ability to take existing workflows and quickly scale them to much larger applications.
+\item Xarray is the interface for working with big datasets: it provides a Pandas-like API for labelled n-dimensional arrays and has backends for established and upcoming self-describing community data file formats and access protocols like netCDF, GeoTIFF, OPeNDAP, and Zarr.  Xarray transparently integrates Dask arrays and hence enables users to easily scale their work to massively parallel computations.
```

Comment:
Do we want references for netCDF, GeoTIFF, OPeNDAP and Zarr?

... enables users to ~~easily~~ scale their work ...

Reply:
I think it's important to name the standard Xarray complies with.

And for _easily_, this paper is also for promoting Pangeo ecosystem, but I'm happy to remove it.